eo-runtime/src/main/eo/org/eolang/bool.eo

```diff
@@ -44,3 +47,11 @@
   # of WHILE is the number of successful cycles. The \rho
   # of the object copied will be set to "while.\rho".
   [f] > while /int
+
+  # Converts this to bytes
+  [] > as-bytes
+    if. > @
+      ^
+      00-00-00-00-00-00-00-01
```

Comment:
@andreoss I think bool must map to one byte, not eight

Reply:
@yegor256 There is no way to do that without introducing narrowing casting to the language. 
`TRUE eq 1` and `TRUE eq 01-`  would mean that `01- eq 1`, which is not since `1` is 8 bytes.

Reply:
@yegor256 Or `bytes` could ignore leading zeros when comparing bytes.

Reply:
@andreoss this should be true: `TRUE eq 01-`, but this one false: `TRUE eq 1`